ironfish-cli/package.json

```diff
@@ -26,18 +26,19 @@
     "@oclif/test": "2.1.0",
     "@types/blessed": "0.1.17",
     "@types/inquirer": "8.2.5",
-    "@types/node": "18.11.16",
+    "@types/node": "20.9.0",
```

Comment:
Does this assume that host machine will be running node v20?

Reply:
Let me see if it still works if I revert this actually -- I'm not sure if we needed to bump that or not

Reply:
Nice, looks like this still works when I roll back the change. Going to resolve this